app/package.json

```diff
@@ -11,9 +11,11 @@
   },
   "scripts": {
     "electron-rebuild": "node -r babel-register ../internals/scripts/ElectronRebuild.js",
-    "postinstall": "npm run electron-rebuild"
+    "postinstall": "yarn electron-rebuild"
   },
+  "license": "MIT",
```

Comment:
Can we replace this line to;
`"license": "GPL-3.0-only",`

or just delete it if not necessary?